docs/02.API-REFERENCE.md

```diff
@@ -2691,6 +2732,51 @@ jerry_resolve_or_reject_promise (jerry_value_t promise,
 - [jerry_release_value](#jerry_release_value)
 - [jerry_value_is_error](#jerry_value_is_error)
 
+# Functions for symbols
+
+These APIs all depend on the ES2015-subset profile.
+
+## jerry_symbol_to_string
+
+**Summary**
+
+Call the SymbolDescriptiveString ecma builtin operation on the api value.
+
+*Note*: Returned value must be freed with [jerry_release_value](#jerry_release_value) when it
+is no longer needed.
+
+**Prototype**
+
+```c
+jerry_value_t
+jerry_symbol_to_string (const jerry_value_t value);
+```
+
+- `value` - symbol value
+- return value
+  - converted string value, if success
+  - thrown error, otherwise
+
+**Example**
```

Comment:
Would it be possible to have fully compilable, runnable examples? We have doc testing in place. That way we can catch if the API changes but the docs do not. (Not just here but in every case in this PR).